src/user-icon/user-icon.scss

```diff
@@ -92,8 +92,8 @@
 
     &::after {
       content: '';
-      width: calc(100% / 1.5);
-      height: calc(100% / 1.5);
```

Comment:
This definitely helps with the centering and looks better in the smaller status icons, but appears a little bit smaller in the larger status icons.  

I tried replacing with `width: $iui-icons-small;` but it looked bugged in the smaller sizes:
![Screen Shot 2022-03-22 at 5 04 21 PM](https://user-images.githubusercontent.com/849817/159575769-3c73f4f1-2900-4ae0-91c9-fd69a5eef8ec.png)

I tried some combination of `calc` and `round` but wasn't having any success there either.  Without setting the width and height explicitly for each size, this may be the only approach I can think of atm.

Reply:
I also do not really like making status icon smaller.. It looks too small on bigger user icon.